cordformation/src/test/resources/net/corda/plugins/DeploySingleNodeWithExternalDBSettingsWithDocker.gradle

```diff
@@ -21,5 +21,8 @@ task prepareDockerNodes(type: net.corda.plugins.Dockerform, dependsOn: 'generate
         cordapps = ["$corda_group:corda-finance-contracts:$corda_release_version",
                     "$corda_group:corda-finance-workflows:$corda_release_version"]
     }
+
     dockerConfig = postgres
+
+    dockerImage = docker_image_name
```

Comment:
Is this the correct Docker image for use with an external database? I see you have removed a reference to this image from the plugin:
```
entdocker.software.r3.com/corda-enterprise-java1.8-4.3
```
I had been assuming that this image has extra added "databaseyness". However, if it is just the basic Docker image for Corda Enterprise 4.3 then what you have here is already fine, with no action required.

FYI the Corda Gradle Plugins are Open Source, and are used equally by both Corda Enterprise and Corda OS. They therefore should not have Enterprise-specific default values - something I probably should have picked up on when I reviewed the previous PR.

Reply:
That is just a basic Docker image for Corda Enterprise 4.3. Also, in the last commit I changed the Dockerform slightly to remove as everything ENT specific. It is now up to the user to provide the correct configuration and the correct Docker image